packages/xstate-test/test/plans.test.ts

```diff
@@ -74,4 +98,51 @@ describe('testModel.testPlans(...)', () => {
       ]
     `);
   });
+
+  describe('When the machine only has one path', () => {
```

Comment:
This test file feels a little thin, lmk if there's anything that you think might need adding.